crypto/dsa/dsa_asn1.c

```diff
@@ -32,6 +32,15 @@ void DSA_SIG_get0(BIGNUM **pr, BIGNUM **ps, const DSA_SIG *sig)
     *ps = sig->s;
 }
 
+int DSA_SIG_set0(BIGNUM *r, BIGNUM *s, DSA_SIG *sig)
```

Comment:
I've created https://github.com/openssl/openssl/pull/1197 which reorders the arguments, so you can do so here, too.